cmd/skaffold/app/cmd/config/set.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func NewCmdSet(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set",
+		Short: "Set a value in the global skaffold config",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			resolveKubectlContext()
+			err := setConfigValue(args[0], args[1])
+			if err == nil {
```

Comment:
might be more idiomatic to do

```
if err != nil {
  return err
}
logConfigChangeForUser(out, args[0], args[1])
```

Reply:
I actually had this before and changed it, because it's one less line to do it this way: the way you described you also need to do a `return nil` afterwards. curious to hear your opinion on this

Reply:
slight preference for the return nil, but up to you